linkatos/message.py

```diff
@@ -1,10 +1,10 @@
 import re
 
+
 def extract_url(message):
-    """
-    Returns the first url in a message. If there aren't any returns None
-    """
-    answer = re.search("(?:\s|^)<(https?://[\w./?+&+%$!#=\-_]+)>(?:\s|$)", message)
+    # Returns the first url in a message. If there aren't any returns None
```

Comment:
It was the only docstring in the whole application and I didn't have a reason for the difference.
Is there a common practice regarding this?